documentation/adr/0001-recording-architecture-decisions.md

```diff
@@ -0,0 +1,79 @@
+# Recording Architecture Decisions
+
+- **Status:** Proposed
+- **Last Modified:** 2023-06-07
+- **Related Issue:** TODO
+- **Deciders:** TODO
+- **Tags:** ADRs
+
+## Context and Problem Statement
+
+Important decisions about the structure of a codebase, the tools or platforms adopted, and the scope or vision of a product are made all throughout a project's lifecycle. Often these decisions are made and changed blindly with little record of why that decision was made, making it difficult for other collaborators and even the original decision makers to understand the justification behind that decision in the future.
+
+_What is the best way to document key architectural decisions made within the project so that future contributors can understand the justification for those decisions?_
+
+## Decision Drivers <!-- RECOMMENDED -->
+
+- **Explicit:** Important decisions about the project architecture should be clear and unambiguous
+- **Documented:** There should be a written record describing which decisions are made
+- **Accessible:** Decision records should be centrally located, searchable, and accessible to non-technical audiences
+- **Portable:** The decision records should be able to travel with the code when it's forked, cloned, etc.
+- **Maintainable:** The system we adopt should be easy to maintain or amend as needed.
+
+## Options Considered
+
+- ADRs in the repo `docs/` folder
+- Comments and docstrings in the code itself
+- Articles in the project Wiki tab
```

Comment:
Should we also consider using the overall wiki we choose for the comms platform?